### PR TITLE
Move com wrapper factory

### PR DESCRIPTION
**Pull request by pharring**
Now that we've updated to a new base drop of VS, the shell implements IComWrapperFactory for us and we can move our BlindAggregatorFactory implementation out of shipping code and into unit tests.

### CODE REVIEW

**Comment by pharring** [comment]
@jasonmalinowski @DustinCampbell for review.

File: src/EditorFeatures/TestUtilities/BlindAggregatorFactory.cs [hunk 1]
```diff
@@ -3,14 +3,13 @@
 using System;
 using System.Runtime.InteropServices;
 using System.Threading;
-using Microsoft.VisualStudio.Shell.Interop;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.Interop
+namespace Microsoft.CodeAnalysis.Test.Utilities
```
Comment (jasonmalinowski) [inline src/EditorFeatures/TestUtilities/BlindAggregatorFactory.cs]: Just curious, what's still using these in tests? The CodeModel tests?

**Comment by jasonmalinowski** [comment]
:+1:

File: src/EditorFeatures/TestUtilities/BlindAggregatorFactory.cs (hunk 1 above)
Comment (pharring) [inline src/EditorFeatures/TestUtilities/BlindAggregatorFactory.cs]: Yes, CodeModel tests.

**Pull request merged**